threeplay.go

```diff
@@ -36,6 +38,18 @@ type File struct {
 	ErrorDescription     string `json:"error_description"`
 }
 
+type FilesPage struct {
```

Comment:
what's the common thing to do when we start having a bunch of structs on the file, we keep it here or move somewhere else?